tech-far-hub/content/resources/index.mdx

```diff
@@ -1,8 +1,14 @@
 ---
 slug: index
 heading: Resources
-template: default
+template: resources-landing
 page_type: landing
 ---
-
-Resources to come
+<ExternalResources heading="Learning Center" media="/assets/img/manik-roy-pYYpgTClGgI-unsplash.jpg">
```

Comment:
is the hash in the jpg name something that is created by Gatsby?

Reply:
it's just from unsplash